src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/edit/details/main.js

```diff
@@ -163,7 +163,7 @@ define([
                     '/admin/api/media/' + curMediaId + '?action=new-version' :
                     '/admin/api/media?collection=' + this.formOptions.accountAvatarCollection;
                 
-                url = url + '&locale=' + encodeURIComponent(this.sandbox.sulu.user.locale);
+                url = url + '&locale=' + encodeURIComponent(this.sandbox.sulu.getDefaultContentLocale());
```

Comment:
It does not always return the first content locale. If the system locale of the current user is also a content locale, the current system locale is returned. But your right, a better solution would be to save the default content locale for a user.